src/sidebar/services/groups.ts

```diff
@@ -464,6 +468,15 @@ export class GroupsService {
       // Persist this group as the last focused group default
       this._store.setDefault('focusedGroup', newGroupId);
     }
+
+    if (
+      // For now, group members are going to be used for at-mentions only, so we
+      // can skip loading them if the feature is not enabled
+      this._store.isFeatureEnabled('at_mentions') &&
+      this._store.getFocusedGroupMembers() === null
+    ) {
+      this._loadFocusedGroupMembers(groupId).catch(e => console.error(e));
```

Comment:
This starts an async process to fetch focused group members, but this request is not canceled if the focused group is changed while it is in-flight. In general, code for dealing with async requests should assume:

- Requests can take an arbitrary amount of time (up to any explicit timeout)
- Requests may not finish in the order they are started

Reply:
Yeah, good point.